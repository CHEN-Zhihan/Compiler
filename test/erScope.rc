a = 1;
b = 2;
if (a > 5) {
    c = 5;
    b = b + c;
} else {
    c = 6;
    b = b + c;
}
puti(b);
puti(c);