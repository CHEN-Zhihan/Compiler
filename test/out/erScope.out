Undefined reference to variable c
